src/test/java/homeworks/POM/Pages/AllSongsPageHmw.java

```diff
@@ -0,0 +1,27 @@
+package homeworks.POM.Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+
+public class AllSongsPageHmw extends BasePageHmw{
+    By shufflebuttonLocator= By.cssSelector(".btn-shuffle-all");
+    By firstSongLocator=By.cssSelector("tr.song-item");
+
+
+    public AllSongsPageHmw (WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    public void Shuffle() {
+driver.findElement(shufflebuttonLocator).click();
+    }
+    public void clickFirstSong(){
+        actions.doubleClick(driver.findElement(firstSongLocator));
+
+    }
+
+
+
```

Comment:
**Please delete the empty line!** "An Empty Line is a "Code Smell" 

The short answer is, They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce.